include/miopen/miopen.h

```diff
@@ -7732,6 +7733,61 @@ MIOPEN_EXPORT miopenStatus_t miopenPReLUBackward(miopenHandle_t handle,
 // CLOSEOUT RELU DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+
+// Any APIs
+/** @addtogroup any
+ *
+ *  @{
+ */
+
+/*! @brief Helper function to query the minimum workspace size required by the any call
+ *
+ * @param handle                   MIOpen Handle (input)
+ * @param inputDesc                Tensor descriptor for data input tensor x (input)
+ * @param dim                      The dimension(s) to reduce (input)
+ * @param keepdim                  Whether the output tensor has dim retained or not (input)
+ * @param outputDesc               Tensor descriptor for output tensor (input)
+ * @param sizeInBytes              Pointer to data to return the minimum workspace size
+ * @return                         miopenStatus_t
+ */
+
+MIOPEN_EXPORT miopenStatus_t
+miopenGetAnyForwardWorkspaceSize(miopenHandle_t handle,
+                                 const miopenTensorDescriptor_t inputDesc,
+                                 const int32_t dim,
+                                 const bool keepdim,
+                                 const miopenTensorDescriptor_t outputDesc,
+                                 size_t* sizeInBytes);
+
+/** @brief Execute an any forward layer
+ *
+ * @param handle                   MIOpen handle (input)
+ * @param workspace                Address of the allocated workspace data (input)
+ * @param workspaceSizeInBytes     Size in bytes of the allocated workspace data (input)
+ * @param inputDesc                Tensor descriptor for input tensor (input)
+ * @param input                    Data tensor input (input)
+ * @param dim                      The dimension(s) to reduce (input)
```

Comment:
I chose `int32_t` for the `dim` parameter simply because other existing examples seem to do so. And I don't see any problem to use `int64_t` if there are benefits to doing so. Are there any previous discussions about the choice between `int32_t` and `int64_t` for `dim` parameters? What are the benefits of using `int64_t` beyond its ability to handle larger numbers? Is there any other additional aspects should I be aware of?